rosdistro_reviewer/submitter/github.py

```diff
@@ -16,9 +16,30 @@
 GITHUB_TOKEN_ENVIRONMENT_VARIABLE = EnvironmentVariable(
     'GITHUB_TOKEN', 'Authentication token secret for GitHub')
 
-INTRODUCTION = """Hi there, thanks for contributing to ROS!
+INTRODUCTION = """Thanks for sending a pull request to ROS distro!
 
-This is an automated review generated by the [rosdistro-reviewer tool](https://github.com/ros-infrastructure/rosdistro-reviewer). If you'd like the tool to run again, just [re-request review from GitHub Actions](https://github.com/ros-infrastructure/rosdistro-reviewer?tab=readme-ov-file#running-rosdistro-reviewer-in-github-actions).
+This is an automated tool that helps check your pull request for correctness. This tool checks a number of attributes associated with your ROS package and generates a report that helps our reviewers merge your pull request in a timely fashion. Here are a few things to consider when sending adding or updating a package to ROS Distro. ROS Distro includes a very helpful [CONTRIBUTING.md](https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md) file that we recommend reading if it is your first time submitting a package. Please also read the [ROS Distro review guidelines](https://github.com/rosdistro/rosdistro/blob/master/REVIEW_GUIDELINES.md) which summarizes this release process.
+
+If you'd like to run this tool again to generate a new review for your PR, just [re-request review from GitHub Actions](https://github.com/ros-infrastructure/rosdistro-reviewer?tab=readme-ov-file#running-rosdistro-reviewer-in-github-actions).
+
+# ROS Distro Considerations
+* ROS Distribtutions are created using [REP-134 Standards Track](https://ros.org/reps/rep-0143.html) as a guide.
+* Your package name should comply to [REP-144 ROS Package Naming](https://www.ros.org/reps/rep-0144.html)
+* Your pacakge must build for all platforms and architectures on the ROS buildfarm. See [REP-2000 ROS Releases and Supported Platforms](https://www.ros.org/reps/rep-2000.html) for all supported platforms for your ROS Distro.
+* Your package must contain an [OSI approved license](https://opensource.org/licenses). Your `package.xml` file must also include that license in a machine readable format. See [REP-149 Package Manifest Format Three Specification](https://ros.org/reps/rep-0149.html#license-multiple-but-at-least-one) for additional details.
+* Your package must include a copyright at the top of every file.
+* A publicly available, open source, repository for your ROS package.
+* While not required, we recommend that you create an account for ROS Discourse and subscribe to the [appropriate release topic](https://discourse.ros.org/c/release/16).
+* If you real time chat would help you resolve an issue please join our [Discord Server] and join the `#infra-help` channel.
```

Comment:
The Discord link is missing.

```suggestion
* If you would like, you may join our [Discord Server](https://discord.com/servers/open-robotics-1077825543698927656) and ask questions in the `#infra-help` channel.
```

I feel slightly hesitant to promise real time support.

Reply:
Taken verbatim in f4f160f.